docs/source/migration/3.0.mdx

```diff
@@ -47,18 +47,6 @@ val client = ApolloClient.builder()
   .build()
 
 ```
-4. Queries and other operations:
-```kotlin
-// Replace:
-val response = apolloClient
-  .query(MyQuery())
-  .await()
-
-// With:
-val response = apolloClient
-  .query(MyQuery())
-  .execute()
-```
```

Comment:
Maybe keep it, even if we have deprecation helpers?

Reply:
Even if we're in the "Quick Route" section?

Reply:
Sorry I missed that this was in the "Quick Route". All good 👍👍